labelImgPlusPlus.py

```
@@ -2408,32 +2952,226 @@ def _auto_save_triggered(self):
+        labels = []
+        base_path = os.path.splitext(img_path)[0]
+
+        # Try XML (PASCAL VOC)
+        xml_path = base_path + XML_EXT
+        if os.path.exists(xml_path):
+            try:
+                reader = PascalVocReader(xml_path)
+                shapes = reader.get_shapes()
+                labels = [shape[0] for shape in shapes]
+            except Exception:
+                pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)